Todo-script.js

```diff
@@ -1,18 +1,60 @@
+const POPUP_ERROR = "<div class='error-popup'>";
+const POPOUP_TIME = 1500;
+const CL_ERROR = "Item not found";
+const CHILL = "You have nothing to do";
+let todoItems = [];
 $(document).ready(() => {
   const SPAN_CLOSE = "<span class='close'>\u00D7</span>";
-  const POPOUP_TIME = 1500;
-  const POPUP_ERROR = "<div class='error-popup'>";
   const ERROR_MESSAGE = "You must enter something on To-do list";
+  const CLEAR_LIST_MESSAGE = "List is clear!";
```

Comment:
All string contants can be on the top of the file, no need to create them on load.